playwright/tests/screenshots/elements.spec.ts

```diff
@@ -27,6 +27,12 @@ test('end', async ({ page }) => {
   await screenshot(editor.element('end:errorEnd'), 'end-error');
 });
 
+test('boundary', async ({ page }) => {
+  const editor = await ProcessEditor.openProcess(page, { file: '/processes/screenshot/boundary.p.json', waitFor: '.sprotty-graph' });
+  await screenshot(editor.element('userTask'), 'boundary-signal', 0);
+  await screenshot(editor.element('boundary:errorBoundaryEvent'), 'boundary-error');
```

Comment:
no strong preference; but I think with the element attached to it the boundary looks most natural. Otherwise BPMN newbies rarely see the difference to normal Event.